changelog/pending/20250417--cli--add-the-no-dangling-references-flag-to-pulumi-schema-check-to-consider-unresolved-references-a-validation-error.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: cli
+  description: Add the `--allow-dangling-references` flag to `pulumi schema check` to consider unresolved references a validation error.
```

```suggestion
  description: Consider unresolved references a validation error; add the `--allow-dangling-references` flag to opt out of this behaviour
```

I think?  Otherwise it sounds like `--allow-dangling-references` makes it a validation error?